apps/cyberstorm-storybook/LinkLibrary.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  LinkLibrary,
+  ThunderstoreLinkProps,
+} from "@thunderstore/cyberstorm/src/components/LinkingProvider";
```

Comment:
These should be exported directly from `index.ts`. Whoever uses the components don't need to know about the internal paths of cyberstorm package.